Tests/Unit/Encodings/PaymentRequestTests.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+
+import LibMobileCoin
+@testable import MobileCoin
+import XCTest
+
+class PaymentRequestTests: XCTestCase {
+
+    static let publicAddress = (try? PublicAddress.Fixtures.Init())!.accountKey.publicAddress
```

Comment:
Lets move this in-line to each test where its needed. You can change it to something like:

`let publicAddress = try PublicAddress.Fixtures.Init().accountKey.publicAddress`

The force-cast can go away, and if it fails for some reason it will show up in the test results.

Reply:
switched to declaring locally in-line